launcher/build.rs

```diff
@@ -0,0 +1,24 @@
+use std::fs::File;
```

Comment:
Can you use [vergen](https://docs.rs/vergen/latest/vergen/) like in the router?
https://github.com/huggingface/text-generation-inference/blob/main/router/build.rs
and then
https://github.com/huggingface/text-generation-inference/blob/main/router/src/server.rs#L636

It is also possible to ask the user to add values returned by GET `/info`.

Reply:
haha I still prefer no dependencies for that kind of things, but since you're already using it.